decidim-elections/docs/docker/bulletin_board/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.4'
+services:
+  db:
+    image: postgres:11
+    environment:
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+  app:
+    image: decidim/decidim-bulletin-board:0.24.1
+    environment:
+      - DATABASE_URL=postgresql://postgres:postgres@db/decidim_bulletin_board_test
+      - RAILS_ENV=test
+      - RAILS_LOG_TO_STDOUT=true
+      - RAILS_SERVE_STATIC_FILES=enabled
+      - SEED=1
+      - IDENTIFICATION_PRIVATE_KEY='{"kty":"RSA","n":"zMXsZpYPKkDlSmezX898y7zNOaJ7ENIN4kj4UhQ95Vm4HlgTpIs2VMMsO0eqynMaOR_G1mXdqbpbaJtXijBe4V8323QwGm6WVAa71E7pDXa5g6-uo5f8GePitN0YER9y2yNQN4uTaNzJiWV2uLBUYfMdj3SIif31YwLULHAOj3B_oleFK8coE_Qr3NzATcYBmsqE8AR4NljxTO6KDmP1SLdf5GBOBhOAIFbnL_Kpj2xkm7MS3hjMVKpiRhqA1UgX5oKZ8ixBv46fNJF0pBsHi3fHNjK9oZzgdx_AI-YFpdE_40-8bh_g9sWzxacqOM2-MdQLHbvRPEVltO3E8tr6I5YWrylcP7l9VD8OJeqjq2qFYHnGYdmLoD2XuXmI9EuBvSb9H4-qcartxZSIQCimKib_fxZvgrG1FSRRhK6YpvIdGv4-G2zfCCRsC4XD80TYI2bf-oYCoy7eU3_eVHFMV2yg4p1Wnuw2Vgq0edPL_bKaV9JvGx7F-U5juxNN0WZR9LzbPl4ReejzN95lyHgbj0nTH_u3bSpZmgJrQF-PwdnPcG46deVjJgUeosrlC4lQxVrRz0GL58BuFunnz2uYDBDrcJCiG60EbdkAFHjOcXU4wrUWATin7je_aqdBXhSnkTafcJAMvL7Y2Ld7vDge8nLqjAVlAi5am3rN0kqKT6M","e":"AQAB","kid":"a8e86f02ca27e1861bfc49e2a9a4614ca9068f8efdb6d42d19d3aab0eb2a31be"}'
+    ports:
+      - 8000:8000
```

Comment:
I already had something else binding 8000, I had to reconfigure the bulletin board ports here and in the secrets.yml. 

As 8000 is a pretty popular port, maybe would be better to document the workaround ?

Reply:
I agree that 8000 is a classic port (like 8080) and it usually is already used. I'd prefer to first discuss it in an issue as we would need to change the default configuration and also I'd like to change it in the Bulletin Board itself. I have opened the issue at https://github.com/decidim/decidim-bulletin-board/issues/303